tests/spread/integration/python3.12/test_core.py

```diff
@@ -0,0 +1,45 @@
+"""
+Tests some of the core Python functionality
+"""
+
+import sys
+import os
+import socket
+import time
+import urllib
+import urllib.request
+
+
+def hello_world():
+    print("Hello, world!")
```

Comment:
I think it might be helpful to have a separate test case for print. It is not obvious that the other tests are printing as part of the test itself, which means somebody might potentially change it to log something or to remove it altogether. It also means that the failure is more localized if print breaks.

Reply:
I would add that we are not asserting that print works because we never check stdout, we are just checking that the function does not crash.

Reply:
agreed. and asserting the stdout could be tricky as we'd probably need additional modules for that (like `StringIO`) which are not present in the core slice

Reply:
what about asserting stdout in the spread test itself using grep?

Reply:
won't be reliable (or even work) cause the spread task is using variants, which means that grep would only work for the `core` variant. We'd need to add a conditional in the test for that, which I don't think is worth the effort. But if you feel strongly about it, I'll add it...it's just a couple of lines

Reply:
Understood, let's just keep it like this for now. I don't think it is worth it, I thought all the slices had a dependency on the "core" that had "print".